sormas-ui/src/main/java/de/symeda/sormas/ui/samples/PathogenTestForm.java

```diff
@@ -208,18 +220,22 @@ protected void addFields() {
 		testTypeTextField = addField(PathogenTestDto.TEST_TYPE_TEXT, TextField.class);
 		FieldHelper.addSoftRequiredStyle(testTypeTextField);
 		DateTimeField sampleTestDateField = addField(PathogenTestDto.TEST_DATE_TIME, DateTimeField.class);
-		sampleTestDateField.addValidator(
-			new DateComparisonValidator(
-				sampleTestDateField,
-				this::getSampleDate,
-				false,
-				false,
-				I18nProperties.getValidationError(
-					Validations.afterDateWithDate,
-					sampleTestDateField.getCaption(),
-					I18nProperties.getPrefixCaption(SampleDto.I18N_PREFIX, SampleDto.SAMPLE_DATE_TIME),
-					DateFormatHelper.formatDate(
-						sample != null ? sample.getSampleDateTime() : (Date) sampleForm.getField(SampleDto.SAMPLE_DATE_TIME).getValue()))));
+		if (environmentSample == null) {
```

Comment:
I would say, the validation should be applied for environment sample test too

Reply:
removed